src/lib/utils/hooks/useGetEntityHeaders.ts

```diff
@@ -17,7 +17,6 @@ export default function useGetEntityHeaders(props: UseGetProfilesProps) {
     const getData = async () => {
       // look at current list of data, see if incoming ids has new data,
```

Comment:
this file is on deleted develop, do you need to run `git rm --cached  src/lib/utils/hooks/useGetEntityHeaders.ts`